packages/fxa-settings/src/components/ConnectAnotherDevicePromo/index.tsx

```diff
@@ -52,14 +54,35 @@ export function ConnectAnotherDevicePromo() {
           data-testid="play-store-link"
           href="https://app.adjust.com/2uo1qc?redirect=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dorg.mozilla.firefox"
         >
-          <img src={playStoreIcon} alt="" />
+          <Localized
+            id="connect-another-play-store-image"
+            attrs={{ alt: true }}
+          >
+            <img
+              src={getStoreImageByLanguages(
```

Comment:
Sorry for not catching this on my first pass, but this loads the SVG without inlining it, you can see the image loaded from the Network tab. I was going to say it's fine for us to do it in a maintenance follow up issue if you don't have time to do that before we tag midday tomorrow since it's not a big deal, but I decided to check into your branch and poke around at this real quick...

https://github.com/mozilla/fxa/compare/translate-appstore-logos...LZoog:translate-appstore-logos-inline?expand=1#diff-8c237250a12d1f150fe5bdc043cb71c45e70c5c03f0935e57aa9575baa3c3c04R144

This is how you should be able to inline these but you'd also need to update the `alt` text in the FTL file to an `aria-label` instead, we have examples of how to do this in a few places. I'm wondering if we're maybe missing something in the image map though, since I get Typescript complaints from line in that link above saying it's possibly undefined. 🤔

If we don't get these inlined here, let's at least make sure that can't come back as `undefined`. Also, I think to follow other patterns, we can just plop your directories of images in this component itself, as in have all of the new SVGs live in like `ConnectAnotherDevicePromo/google-play-store/[whatever].svg`, similar to how we import other SVGs. I think it might also be more consistent to use that kebab case based on our other icon names but I don't care too much about that.